oecophylla/taxonomy/oecophylla-kraken.yaml

```diff
@@ -1,6 +1,5 @@
 channels:
   - bioconda
 dependencies:
-  # - python==2.7
   - kraken
   # - bracken
```

Comment:
Should this be commented out?

Reply:
Tested and working. The real problem is Bracken, which can be automatically installed in local system, but not in Travis. I created issue #83

Reply:
Let's hold off on merging for now until we have this figured out, maybe?